src/PagesBody/ProfilePage/ProfilePage.js

```diff
@@ -1,37 +1,26 @@
 import propTypes from 'prop-types';
-import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 
 import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
 import MainButton from '../../Atoms/MainButton/MainButton';
 import ProfileCard from '../../Molecules/ProfileCard/ProfileCard';
 
 const ProfilePage = ({ userId }) => {
-  const navigate = useNavigate();
-
   return (
     <ColumnWrapper paddingLeftRight={1}>
       <ProfileCard />
-      <MainButton
-        text="Twoje Psy"
-        onClick={() => navigate('/userDogs')}
-        ternary
-      />
-      <MainButton
-        text="Twoje Konkursy"
-        onClick={() => navigate(`./${userId}/contests`)}
-        // onClick={() => navigate('/contests')}
-        ternary
-      />
-      <MainButton
-        text="Twoje Dane"
-        onClick={() => navigate('/userData')}
-        ternary
-      />
-      <MainButton
-        text="Nadchodzące Konkursy"
-        onClick={() => navigate('/upcoming')}
-        secondary
-      />
+      <Link to="/user-dogs">
```

Comment:
This is a case of button inside an anchor tag, and its semantically incorrect an unnecessary. Perhaps creating a styled link that looks like a button would be an option?
Also @pierwszazlewej consult with the UI/UX designer if the picture of the dog should not appear in another place, it doesn't look doog below the app logo. (probably out of scope of this PR)

Reply:
Solved

Reply:
https://www.youtube.com/watch?v=nf0oXY4nDxE
Link looks like a button